Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Contributors to the PostgreSQL Operator Project
+#
+# This Source Code Form is subject to the terms of the Eclipse Public
+# License v. 2.0. If a copy of the EPL was not distributed with this
+# file, You can obtain one at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
```

Comment:
Please, use the same license header as the `.rs` files

Reply:
done